docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[indicators-of-compromise]]
+= Indicators of compromise
+
+The Intelligence page collects data from enabled threat intelligence feeds and provides you with a centralized view of indicators. This topic offers definitions of threat intelligence and indicators, helps you get started using the Intelligence page, and explains how to work with indicators.
+
+[role="screenshot"]
+image::images/indicators-table.png[Shows the Indicators table on the Intelligence page]
+
+[discrete]
+[[ti-indicators]]
+== Threat intelligence and indicators
+If you are unfamiliar with threat intelligence, think of it as a research function that helps an organization make informed decisions when taking actions to protect itself. An organization's security operation center (SOC) team is usually responsible for managing threat intelligence. A SOC team's goals are to understand current and emerging threats so they can recommend actions that will prevent, stop, and mitigate threats to an organization.
+
+An indicator, also referred to as an Indicator of Compromise (IoC), is a piece of information that represents a known malicious threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
+
+[discrete]
+[[setup-intelligence-page]]
+== Set up the Intelligence page
+
+Install threat intelligence integrations to add indicators to the Intelligence page.
+
+IMPORTANT: Before enabling a threat intelligence feed, ensure you have installed a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] on the hosts you want to monitor.
+
+. Choose one of the following:
+.. From the {security-app} main menu, click the Intelligence page, then click *Add Integrations*.
+.. From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter for threat intelligence integrations.
+. Select a threat intelligence integration, then complete the installation steps.
++
+NOTE: Refer to the Threat intelligence section of the https://docs.elastic.co/integrations[Elastic integration documentation] for more information about required field mappings.
```

Comment:
This was a little unclear at first, partly because "Threat intelligence" is only a menu heading on the target page. It might help to be more explicit about this, because I was confused when I first landed on the page.
```suggestion
NOTE: For more information about required field mappings, go to the https://docs.elastic.co/integrations[Elastic integration documentation] and search for a specific threat intelligence integration.
```

Reply:
^This works and sorry for the confusion. I don't think its actually possible to link to the Threat Intelligence section in the integration docs (yet) 😅  Though, do let me know if there's a way. Otherwise, I like the suggestion you made to tell users that they could search for the TI integration they're installing.

Reply:
@nastasha-solomon Yeah I tried to find a way to link to that section too, but couldn't find anything. "Search for it" seemed easier to say than trying to explain how to use the nav menu.